src/jdk.crypto.cryptoki/unix/native/libj2pkcs11/p11_md.c

```diff
@@ -264,18 +264,19 @@ JNIEXPORT jobject JNICALL Java_sun_security_pkcs11_wrapper_PKCS11_connect
  * Method:    disconnect
  * Signature: ()V
```

Comment:
nit: update the JNI method signature? Same for all other p11_md.c for different OS...

Reply:
Yes.